Templates/TraditionalBridge/InterfaceDispatcherData.tpl

```diff
@@ -1,4 +1,4 @@
-<CODEGEN_FILENAME><SMC_INTERFACE>DispatcherCustom.dbl</CODEGEN_FILENAME>
+<CODEGEN_FILENAME><SMC_INTERFACE>DispatcherData.dbl</CODEGEN_FILENAME>
```

Comment:
whats the purpose of this change?

Reply:
0c9a19280f8e5a9e3138ee8e661aed124b62acfd changed back to DispatcherCustom